src/bindings/python/README.md

```diff
@@ -0,0 +1,56 @@
+# OpenVINO Python API
+
+OpenVINO Python API is a part of OpenVINO library. The component is responsible for:
+
+* Bindings of OpenVINO - allowing users to utilize OpenVINO library in their Python code. Python API provides bindings to basic and advanced APIs from OpenVINO Runtime.
+* Extending OpenVINO with pythonic features - on top of direct translations from C++, Python API component:
+    * Adds specific extensions to support numpy-based data.
+    * Provides support for external frameworks inputs.
+    * Provides shortcuts and helpers with more pythonic design.
+    * Allows to apply advanced concepts like shared memory to take full advantage of OpenVINO.
+
+## Key contacts
+
+If you have any questions, feature requests or want us to review your PRs, send us a message or ping us on Github.
+
+Team members and main responsibilities:
+
+* [Jan Iwaszkiewicz](https://github.com/jiwaszki) (e-mail: jan.iwaszkiewicz@intel.com)  
+Techincal Lead: questions, issues and feature requests, memory and GIL related questions
+* [Anastasia Kuporosova](https://github.com/akuporos) (e-mail: anastasia.kuporosova@intel.com)  
+Developer: building of the component, general bindings
+* [Przemyslaw Wysocki](https://github.com/p-wysocki) (e-mail: przemyslaw.wysocki@intel.com)  
+Developer: project requirements, general bindings
+
+## Components
+
+```
+openvino/src/bindings/python
```

Comment:
We already exists here :)
Maybe we can use a usual list as we have for other components?

Reply:
I see both options as equal for developers. However, if it is general practice across components, sure I will edit this.

Reply:
Done